spec/features/users/admin/admin_user_index_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe "As an admin" do
+  context "when i visit the admin index page" do
+    before(:each) do 
+        @admin = create(:user, role:2)
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@admin)
+
+        @user1 = create(:user)
+        @user2 = create(:user)
+        @merchant = create(:user, role: 1)
+
+        visit root_path   
+        click_on 'Users'
+    end 
+
+    it "can see all default users" do
+        expect(page).to have_content(@user1.name)
+        expect(page).to have_content(@user2.name)
+        save_and_open_page
```

Comment:
May want to remove save_and_open_page so we all don't hit it when we run RSpec; looks good though!